doc/branding.md

```diff
@@ -72,15 +72,30 @@ With the configuration set as above, the title will be: `TeskaLabs | Page Title`
 
 Every image HAS TO be provided in SVG (vectorized).
 Use of pixel formats (PNG, JPG, ...) is strongly discouraged.
+While creating the branding images, use full width/height of the canvas (ratio 3:1 on full and 1:1 on minimized version). <em>No padding is required for optimal viewing experience.</em>
 
 ### Branding images
 
-Top-left location.
+format: **SVG**
```

Comment:
@petrKavulok Not sure if you took a look how does it look like in some markdown compiler, but this is still not correct. If you want to have it on separate line, you need to add an empty newline below the text :)

Reply:
> @petrKavulok Not sure if you took a look how does it look like in some markdown compiler, but this is still not correct. If you want to have it on separate line, you need to add an empty newline below the text :)

I did check it with this 'markdown preview' extension. 
I don't think I understand what you mean 

<img width="1680" alt="Snímek obrazovky 2022-08-04 v 12 54 14" src="https://user-images.githubusercontent.com/79644454/182830349-e904e05f-0f54-4ec5-87cc-3e6562a420a9.png">

Reply:
@petrKavulok Just take a look here in github preview: https://github.com/TeskaLabs/asab-webui/blob/e62719cc3fbaf4487162513bcd9a44be216feb67/doc/branding.md

if you want to have a newline in markdown, you need to add an empty newline between the rows

```
format: **SVG**

full:
...
```
It seems like a markdown compiler of visualstudio "schroustne vsechno" :)